Architect-Coders/MU8
# Issue #47: Save events room database

**Save events room database** #47 
- The event table was create.
- The counter query, query and insertion of event data was created

**Testing Done:**

- Huawei p20 Lite

**How to test:**

- Run the application and click in events
- You can see the loading while the event have been loaded
- Then you can see the events in a scrollable list
- close the application 
- Disable internet
- Open the application and click in events

**Signed-off-by**:

- Santiago Lozada <santiagolozada95@gmail.com>
